### ISSUE TITLE
multiline-html-element-content-newline case sensitivity

### ISSUE THREAD

**Issue by armano2** [vuejs/eslint-plugin-vue#662]
**Tell us about your environment**	
* **ESLint version:**	5.9.0
* **eslint-plugin-vue version:**	5.0.0.beta4 0fd0f7b0a305429d161abb4d59596beb78897c80
* **Node version:**	10

**Please show your full configuration:**
```json
{
  "eslint vue/multiline-html-element-content-newline": ["error", {
    "ignores": ["VueComponent", "pre", "textarea"]
  }]
}
```

**What did you do?**	

```vue
<VueComponent>multiline
content</VueComponent>

<VueComponent><span
  class="bold">For example,</span>
Defines the Vue component that accepts preformatted text.</VueComponent>
```

**What did you expect to happen?**
There should be no errors

**What actually happened?**
```
[ESLint] Expected 1 line break after opening tag (`<vuecomponent>`), but no line breaks found. (vue/multiline-html-element-content-newline)
...
```
https://armano2.github.io/eslint-plugin-vue/rules/multiline-html-element-content-newline.html#ignores-vuecomponent-pre-textarea

**Comment by armano2** [comment]
we should also check for `IgnoreTag` -> `ignore-tag` matching
